stdlib/3/multiprocessing/context.pyi

```diff
@@ -0,0 +1,113 @@
+# Stubs for multiprocessing.context
+
+# NOTE: These are incomplete!
+
+from logging import Logger
+from typing import Any, Callable, Optional, List, Sequence, Tuple, Union
+
+class BaseContext(object):
+    def cpu_count(self) -> int: ...
+
+    # TODO: change return to SyncManager once a stub exists in multiprocessing.managers
+    def Manager(self) -> Any: ...
+
+    # TODO: change return to Pipe once a stub exists in multiprocessing.connection
+    def Pipe(self, duplex: bool) -> Any: ...
+
+    # TODO: change return to Lock once a stub exists in multiprocessing.synchronize
+    def Lock(self) -> Any: ...
+
+    # TODO: change return to RLock once a stub exists in multiprocessing.synchronize
+    def RLock(self) -> Any: ...
+
+    # TODO: change lock param to Optional[Union[Lock, RLock]] when stubs exists in multiprocessing.synchronize
+    # TODO: change return to Condition once a stub exists in multiprocessing.synchronize
+    def Condition(self, lock: Optional[Any] = None) -> Any: ...
```

Comment:
Think I understand after reading [this](https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values) that it is for all defaults.